datajunction-server/datajunction_server/api/authentication/github.py

```diff
@@ -81,19 +80,19 @@ async def get_access_token(
             http_status_code=HTTPStatus.UNAUTHORIZED,
             errors=[DJError(message=message, code=ErrorCode.OAUTH_ERROR)],
         )
-    token = access_data["access_token"]
-    response = JSONResponse(
-        content={"message": "Successfully logged in through GitHub OAuth"},
-        status_code=HTTPStatus.OK,
+    user = github.get_github_user(access_data["access_token"])
+    response = RedirectResponse(
+        url=settings.frontend_host,
     )
-    jwt = create_jwt({"sub": encrypt(token)})
-    response.set_cookie(key="__dj", value=jwt, httponly=True, samesite="strict")
+    if not user:
+        raise DJException(
+            http_status_code=HTTPStatus.UNAUTHORIZED,
+            errors=DJError(
+                code=ErrorCode.OAUTH_ERROR,
+                message=(
+                    "Could not retrieve user using the GitHub provided access token"
+                ),
+            ),
+        )
+    response.set_cookie(key="__dj", value=encrypt(user.username))
```

Comment:
Nit, it would be nice if `__dj` was a contant.

Reply:
Good idea, done!